Nuake/src/Scripting/Modules/SceneModule.h

```diff
@@ -61,13 +64,52 @@ namespace Nuake {
 				RegisterMethod("BrushGetTargetsCount_(_)", (void*)BrushGetTargetsCount);
 			}
 
+			static void CreateEntity(WrenVM* vm)
+			{
+				const std::string name = wrenGetSlotString(vm, 1);
+				const int entity = Engine::GetCurrentScene()->CreateEntity(name).GetHandle();
+				wrenSetSlotDouble(vm, 0, entity);
+			}
+
 			static void GetEntity(WrenVM* vm)
 			{
 				std::string name = wrenGetSlotString(vm, 1);
 				int handle = Engine::GetCurrentScene()->GetEntity(name).GetHandle();
 				wrenSetSlotDouble(vm, 0, handle);
 			}
 
+			static void AddPrefab(WrenVM* vm)
+			{
+				const std::string& prefabPath = wrenGetSlotString(vm, 1);
+				if (prefabPath.empty())
+				{
+					Logger::Log("[Scripting] Cannot add prefab with an empty path.", CRITICAL);
+					wrenSetSlotDouble(vm, 0, -1); // -1 is an empty entity.
+					return;
+				}
+
+				const std::string& prefabName = wrenGetSlotString(vm, 1);
+				if (prefabName.empty())
+				{
+					Logger::Log("[Scripting] Cannot add prefab with an empty name.", CRITICAL);
+					wrenSetSlotDouble(vm, 0, -1); // -1 is an empty entity.
+					return;
+				}
+
+				if (!FileSystem::FileExists(prefabPath))
+				{
+					Logger::Log("[Scripting] Cannot add prefab. File not found: " + prefabPath, CRITICAL);
+					return;
+				}
+
+				Entity& newEntity = Engine::GetCurrentScene()->CreateEntity(prefabName);
+				auto& prefabComponent = newEntity.AddComponent<PrefabComponent>();
+				prefabComponent.PrefabInstance = Prefab::New(prefabPath);
+
+				wrenSetSlotDouble(vm, 0, newEntity.GetHandle());
+				return;
```

Comment:
since it's a void method, i don't think we need the last `return;` since it will exit anyway

Reply:
Fixed